src/js/helpers/JSONPUtil.js

```diff
@@ -0,0 +1,42 @@
+const JSONPUtil = {
+  /**
+   * Request JSONP data
+   *
+   * @todo: write test to verify this util is working properly
+   *
+   * @param {string} url
+   * @returns {Promise} promise
+   */
```

Comment:
Of course you could do your private todo-comments, but please don't check them in.